lib/generators/activeadmin_async_exporter/templates/admin_report.rb.tt

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Do you believe it is necessary since it works from ruby 2.3?

Reply:
True, this magic sauce works from 2.3 and yes, we'll need it until 3.0! By then, all string will be frozen by default.

Reply:
Cool 👍